steam/http.py

```diff
@@ -209,8 +209,8 @@ async def login(self, username: str, password: str, shared_secret: str | None) -
                     "Cannot perform redirects after login. Steam is likely down, please try again later."
                 )
 
-            for url in resp["transfer_urls"]:
```

Comment:
I'd like the code here to still use the transfer_urls incase steam adds new ones down the road and the change isn't caught.

Reply:
this uses `URL.HELP` and `URL.STORE` over `resp["transfer_urls"]` because `CookieJar` needs the yarl-like attributes and URL has those

Reply:
You can create the yarl.URL yourself from the transfer url by doing `URL(url)`.

Reply:
here is what im referencing
https://github.com/aio-libs/aiohttp/blob/master/aiohttp/cookiejar.py#L138

Reply:
> You can create the yarl.URL yourself from the transfer url by doing `URL(url)`.

sure i should be able to do that